### PR TITLE
Allow component text to be set via props to facilitate translation

### PR DESCRIPTION
**Pull request by mklewandowski**
This PR updates the new DESE components so any component label text can be set via props.  This is to help facilitate translation and allow any user-facing text tp be specified by the parent.

### CODE REVIEW

File: src/stories/06-save-buttons.storie.tsx [hunk 1]
```diff
@@ -14,3 +14,9 @@ export const clickableSaveButton = () =>
     onClick={()=>{window.alert("Save")}}
   />;
 
+export const labeledSaveButton = () =>
```
Comment (kswenson) [inline src/stories/06-save-buttons.storie.tsx:17]: Unrelated to this PR, but shouldn't the name of this file be *.stories.tsx?

**Review by kswenson — APPROVED** [review]
👍 LGTM -- one unrelated comment about file naming.

File: src/stories/06-save-buttons.storie.tsx (hunk 1 above)
Comment (mklewandowski) [reply src/stories/06-save-buttons.storie.tsx:17]: Ah, yes!  Thanks!

Comment (mklewandowski) [reply src/stories/06-save-buttons.storie.tsx:17]: Fixed.

**Pull request merged**